### DISCUSSION TITLE
Default --paper-tabs-selection-bar-color to --app-header-text-color

### DISCUSSION THREAD

**Discussion by KTibow** [home-assistant/frontend#7946] [Theming] [answered]
Right now, if you set the background (`--app-header-background-color`) to white, and the text color (`--app-header-text-color`) to black, the bar under the selected tab is white. Ideally it would be set to `--app-header-text-color` by default.

**Comment by spacegaier** [comment, marked as answer]
Will be fixed by PR https://github.com/home-assistant/frontend/pull/8227.
